site/docs-md/getting-started/with-ionic.md

```diff
@@ -1,59 +1,65 @@
 # Using Capacitor with Ionic
 
 ## Install Capacitor into an Ionic project
+Capacitor is easily installed directly into any Ionic project (1.0-4.x+).
 
-Capacitor installs directly into any Ionic project (1.0-4.x+) with a simple `npm install`:
-
-### Start Ionic Project
+### New Ionic Project
 
 ```bash
-ionic start myApp tabs
+ionic start myApp tabs --capacitor
 cd myApp
 ```
 
-Select "N" when asked to add Cordova.
-
-### Build your Ionic App
-
-You must run `npm run build` at least once to create the `www` folder:
+### Existing Ionic Project
 
 ```bash
-npm run build
+cd myApp
+ionic integrations enable capacitor
 ```
 
-### Install Capacitor
+### Initialize Capacitor with your app information
 
-Next, install capacitor into your project:
-
-```bash
-npm install --save @capacitor/cli @capacitor/core
-```
-
-### Init Capacitor with your app information
+*Note: `npx` is a new utility available in npm 5 or above that executes local binaries/scripts to avoid global installs.*
 
 ```
 npx cap init [appName] [appId]
 ```
 
 where `appName` is the name of your app, and `appId` is the domain identifier of your app (ex: `com.example.app`).
 
+*Note: This can be ran any time you would like to update the app information.*
+
+### Build your Ionic App
+
+You must build your Ionic project at least once before adding any native platforms.
+
+```bash
+ionic build
+```
+
+This creates the `www` folder that Capacitor has been [automatically configured](/docs/basics/configuring-your-app) to use as the `webDir` in `capacitor.config.json`.
+
 ### Add Platforms
 
 ```bash
 npx cap add ios
 npx cap add android
 ```
 
-### Open IDE to build
+Both `android` and `ios` folders at the root of the project are created. These are entirely separate native project artifacts that should be considered part of your Ionic app (i.e., check them into source control, edit them in their own IDEs, etc.).
+
+### Open IDE to build, run, and deploy
 
 ```bash
 npx cap open ios
 npx cap open android
 ```
 
+The native iOS and Android projects are opened in their standard IDEs (Xcode and Android Studio, respectively).
```

Comment:
as the title says "run, and deploy", maybe mention you use the IDEs to run and deploy the apps?